src/components/FloatingLabel/theme.ts

```diff
@@ -0,0 +1,45 @@
+import { FlowbiteFloatingLabelTheme } from '~/src/components/FloatingLabel/FloatingLabel';
+
+export const floatingLabelTheme: FlowbiteFloatingLabelTheme = {
```

Comment:
Check the pattern of the Button component:

https://github.com/themesberg/flowbite-react/blob/main/src/components/Button/theme.ts

instead of having `filled_success`, `filled_error_sm`, etc, use nested definitions. Also abstract the common classes to `base` properties, etc